File: Templates/rust/salvo_sea_orm/entity.rs
<#assign PascalCaseName=NameUtil.toPascalCase(table.getRawName())>
<#assign SnakeCaseName=NameUtil.toSnakeCase(table.getRawName())>
<#assign DbStructData=table.getParent()>

#region config
fileName=${SnakeCaseName}.rs
dir=entity
#endregion

use salvo::prelude::Extractible;
use sea_orm::entity::prelude::*;
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel, Extractible, Deserialize, Serialize)]
#[sea_orm(table_name = "${table.getRawName()}"<#if DbStructData.hasSchema()>, schema_name = "${DbStructData.getRawName()}"</#if>)]
pub struct Model {
    <#list columns as column >

    #[sea_orm(column_name = "${column.getRawName()}" <#if column.hasPrimaryKey()>, primary_key</#if>)]
    pub ${NameUtil.toSnakeCase(column.getRawName())}: ${column.getMapperType()},

    </#list>
}

#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
pub enum Relation {}

impl ActiveModelBehavior for ActiveModel {}

File: Templates/csharp/hotchocolate_freesql/Entity.cs
<#assign DbStructData=table.getParent()>
#region config
fileName=${NameUtil.toPascalCase(table.getRawName())}.cs
dir=Entity
#endregion
using FreeSql.DataAnnotations;


/// <summary>
/// ${table.getRawComment()}
/// </summary>
[Table(Name = "<#if DbStructData.hasSchema()>${DbStructData.getRawName()}.</#if>${table.getRawName()}")]
public partial class ${NameUtil.toPascalCase(table.getRawName())} {

<#list columns as column >
    /// <summary>
    /// ${column.getRawComment()}
    /// index:
    /// </summary>
    [Column(Name = "${column.getRawName()}" <#if column.hasPrimaryKey()>, IsPrimary=true</#if>)]
    public ${column.getMapperType()} ${NameUtil.toPascalCase(column.getRawName())} {get;set;}
</#list>

}
